### ISSUE TITLE
Very confused by installation intructions

### ISSUE THREAD

**Issue by ericdgit** [dtylman/gowd#7]
Hey!

I want to use NW.js in my Go project, but I have absolutely no experience from installing it. I am very confused by the steps in your installation instruction. Would be great if you could write an ELI5 version of it.

Cheers!

**Comment by dtylman** [comment]
Hi, 

I am afraid I don't have the capacity to do ELI5 version of the installation.

Maybe you want to start here: http://docs.nwjs.io/en/latest/For%20Users/Getting%20Started/

Good luck!
